erpnext/patches/v12_0/create_banks_for_bank_name_in_employee_and_salary_slip.py

```diff
@@ -0,0 +1,14 @@
+import frappe
+
+
+def execute():
+	bank_names = frappe.db.sql_list("SELECT DISTINCT bank_name FROM `tabEmployee` WHERE bank_name IS NOT NULL") + \
```

Comment:
`IS NOT NULL` is not enough..
You'll also have to check for empty string
or you can combine the 2 conditions as `ifnull(bank_name, '') != ''`